hypha/static_src/src/sass/apply/custom/_custom.scss

```diff
@@ -1 +1,10 @@
 // stylelint-disable no-empty-source
+// stylelint-disable selector-class-pattern
+
+.reviews-summary__td {
+    text-align: center;
+}
+
+.counts-separator {
+    color: $color--mid-grey;
+}
```

Comment:
`_custom.scss` should be empty in stock hypha. It is used to add your custom css without worrying about merge conflicts.